CLI/commands/investor_portal.js

```diff
@@ -498,8 +498,8 @@ async function investCappedSTO(currency, amount) {
     if (raiseTypes[0] == 'POLY') {
         let userBalance = await polyBalance(User.address);
         if (parseInt(userBalance) >= parseInt(cost)) {
-            let allowance = await polyToken.methods.allowance(STOAddress, User.address).call();
-            if (allowance < costWei) {
+            let allowance = await polyToken.methods.allowance(User.address, STOAddress).call();
+            if (parseInt(allowance) < parseInt(costWei)) {
```

Comment:
This is a question rather than a change request, again it's something that caught me out when I was working on a CLI for a STO contract I'm creating. In multiple CLI files there were less than or greater than comparators of number strings. In my CLI's I did like you've done and parsed them to integers or float as a quick fix but I was wondering if some of them at least should be compared as BN instead of integers/float?